openmetadata-ui/src/main/resources/ui/src/components/common/DomainLabel/DomainLabel.component.tsx

```diff
@@ -142,13 +143,15 @@ export const DomainLabel = ({
               width={16}
             />
             {domainLink}
-          </Space>
+          </div>
         </>
       );
     }
 
     return (
-      <Space data-testid="header-domain-container">
+      <div
+        className="d-flex items-center gap-1"
+        data-testid="header-domain-container">
         <Typography.Text className="self-center text-xs whitespace-nowrap">
```

Comment:
why we are removing antd's component and adding div?

Reply:
Antd space add 2 more extra div in the DOM which further create issue with icon and text misalignment. That the reason we are avoding it